conda/recipes/rmm/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
 
 build:
   number: {{ GIT_DESCRIBE_NUMBER }}
-  string: cuda_{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  string: cuda{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}_{{ cudaMallocAsync }}
```

Comment:
Is the `cudaMallocAsync` bit still needed here given the changes above and the fact it is implicit based on CUDA version?

Reply:
We have some logic below that depends on this extra metadata in the build string. If you know a way we can work around that, then we could remove the extra `{{ cudaMallocAsync }}` bit here.

https://github.com/rapidsai/rmm/blob/76ae62207bd083dfd43b7f755f2934458dd7d8bc/ci/gpu/build.sh#L116-L123